build.sbt

```diff
@@ -2,8 +2,8 @@ lazy val V = _root_.scalafix.sbt.BuildInfo
 
 lazy val scala2Version      = V.scala213
 lazy val scala3Version      = "3.1.3"
-lazy val rulesCrossVersions = Seq(V.scala213)
-lazy val allVersions        = rulesCrossVersions :+ scala3Version
+lazy val rulesCrossVersions = Seq(V.scala213, scala3Version)
+lazy val allVersions        = rulesCrossVersion
```

Comment:
I don't actually think this will work ...

Reply:
eh, I don't really know what I'm doing